src/components/entity-info.component.js

```diff
@@ -124,7 +124,7 @@ export const EntityInfo = ({ entity, orgs, language, navigation }: Props) => {
           subtitle={entity.email}
           subtitleStyle={styles.listSubTitle}
           onPress={() =>
-            Communications.email([entity.email], null, null, 'Hi!', '')}
+            Communications.email([entity.email], null, null, null, '')}
```

Comment:
While not technically under the scope of this PR, could you change the last value to `null`, too, instead of an empty string? They both work, but the react-native-communications documentation says `null` is the way to go.